File: src/main/java/tc/pbkk/servistukangantar/model/Delivery.java
package tc.pbkk.servistukangantar.model;

import javax.persistence.*;
@Entity
public class Delivery {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="id")
	private Integer id;	
	private Integer orderId;
	private Integer driverId;
	private String status;
	private Long sentAt;
	private Long arrivedAt;

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public Integer getOrderId() {
		return orderId;
	}

	public void setOrderId(Integer orderId) {
		this.orderId = orderId;
	}

	public Integer getDriverId() {
		return driverId;
	}

	public void setDriverId(Integer driverId) {
		this.driverId = driverId;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public Long getSentAt() {
		return sentAt;
	}

	public void setSentAt(Long sentAt) {
		this.sentAt = sentAt;
	}

	public Long getArrivedAt() {
		return arrivedAt;
	}

	public void setArrivedAt(Long arrivedAt) {
		this.arrivedAt = arrivedAt;
	}
}

File: app/DeliveryService/app/src/main/java/id/gsculerlor/deliveryservice/ui/model/Delivery.kt
package id.gsculerlor.deliveryservice.ui.model

import com.google.gson.annotations.SerializedName

data class Delivery(

    @SerializedName("id")
    val id: Int? = null,

    @SerializedName("order_id")
    val orderId: Int? = null,

    @SerializedName("driver_id")
    val driverId: Int? = null,

    @SerializedName("status")
    val status: String? = null,

    @SerializedName("sent_at")
    val sentAt: Long? = null,

    @SerializedName("arrive_at")
    val arriveAt: Long? = null
)